jobs/models.py

```diff
@@ -1,3 +1,62 @@
+from django.utils import timezone
+
 from django.db import models
+from django_countries.fields import CountryField
+from djangogirls import settings
```

Comment:
import the django conf settings, not the local settings
from django.conf import settings
(https://docs.djangoproject.com/en/1.7/topics/settings/)